### PR TITLE
fix: Added styles to datagrid drag and drop pattern

### PR DESCRIPTION
**Pull request by kai-liu**
Contributes to #1890 

Added styles  that the drag and drop pattern is identical to the WKC data grid component. Additionally, changed element class names in `draggableElement.js` to be consistent to the naming conventions 

#### What did you change?

`draggableElement.js` changed element class names
`draggableElement.scss` added styles
`cspell.json` added accepted word 'draggable'

#### How did you test and verify your work?

Verified via storybook that the style meets requirement.

### CODE REVIEW

File: packages/cloud-cognitive/src/components/Datagrid/Datagrid/DraggableElement.js [hunk 1]
```diff
@@ -11,10 +11,12 @@ import PropTypes from 'prop-types';
 import { Draggable16 } from '@carbon/icons-react';
 import { useDrag, useDrop } from 'react-dnd';
 import cx from 'classnames';
+import { pkg } from '../../../settings';
 
 const { useEffect, useRef, useState } = React;
+const blockClass = `${pkg.prefix}--datagrid`;
 
-const DRAG_TYPE = 'wkc-shared-ui-draggable-element';
+const DRAG_TYPE = `${blockClass}__shared-ui-draggable-element`;
```
Comment (holmansze) [inline packages/cloud-cognitive/src/components/Datagrid/Datagrid/DraggableElement.js:19]: I can't find a corresponding class name in `draggableElement.scss`. Is the style defined somewhere else?

**Review by holmansze — COMMENTED** [review]
Looks good to me!
Just a tiny question if you could take a look. Thanks.

**Review by holmansze — APPROVED** [review]
@kai-liu clarified that the `wkc-shared-ui-draggable-element` style is used by the code. So it should be kept.

**Review by matthewgallo — APPROVED** [review]
Looks great @kai-liu 🎉

**Pull request merged**